qiskit/visualization/state_visualization.py

```diff
@@ -35,7 +35,7 @@
 from .exceptions import VisualizationError
 
 
-@deprecate_arguments({"rho": "state"})
+@deprecate_arguments({"rho": "state"}, since="0.15.1")
 @_optionals.HAS_MATPLOTLIB.require_in_call
```

Comment:
Oh boy we need to clear those out haha.  I'm also concerned if they got deprecated in an actual patch release!

Reply:
I copied this from @1ucian0's PR - I'm not sure how he got 0.15.1, but I imagine git blame.